capstone/scripts/convert_s3.py

```diff
@@ -0,0 +1,403 @@
+import base64
+import boto3
+import hashlib
+import json
+import requests
+import tempfile
+from botocore.exceptions import ClientError
+from collections import namedtuple
+
+from capapi.documents import CaseDocument
+from capapi.serializers import NoLoginCaseDocumentSerializer, CaseDocumentSerializer
+from capdb.models import Reporter, VolumeMetadata, CaseMetadata
+
+s3_client = boto3.client("s3")
+
+
+# Doing this with the API because there are many fewer than cases
+def put_volumes_reporters_on_s3(redacted):
+    bucket = get_bucket_name(redacted)
+
+    for file_type in ["reporters", "volumes"]:
+        current_endpoint = f"https://api.case.law/v1/{file_type}/"
```

Comment:
Another minor thing, which could help with testing:
There are multiple functions in this file in which `https://api.case.law/v1/` is hardcoded. 
Hoisting this into a variable which can be replaced at runtime could help with testing. For example, you could decide to make it point to a different host (local?) which hosts mock files.